src/Table.luna

```diff
@@ -126,21 +273,102 @@ class Table:
         self.fromWrapper $ self.ptr.filter pred
+    
+    # Returns `Table` value with columns for which the given predicate returns `True`.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = ["a", "b", "c", "d", "e"]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > filteredTable = table.filterColumns .isNumeric
+    #
+    # `f`: The predicate (Boolean-valued function).
+    # `return`: Table with columns satisfying predicate.
+
     def filterColumns pred:
         cols = self.toList
         filtered = cols.filter pred
         self.fromColumns filtered
+    
+    # Returns `Table` value with results of applied functions on each of the columns from `self` table.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > filteredTable = table.eachColumn .max
+    #
+    # `pred`: {min, max, mean, median, std, var, sum}
+    # `return`: Table with function application results.
+
     def eachColumn pred:
         cols = self.toList
         filtered = cols.each pred
         self.fromColumns filtered
+
+    # Returns `Column` value with results of function `f` application.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > column = table.each (_.at "col1" * 10)
+    #
+    # `f`: Function to apply on `self` table.
+    # `return`: `Column` value with function application results.
+
     def each f:
         mapper = f TableHandle
         columnWrapper = self.ptr.eachToColumn "Result" mapper
         Column.fromColumnWrapper columnWrapper
+    
+    # Adds to `self` table a column with `name` and values as a result of application function `f` on the `self` table.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > newTable = table.eachTo "col3" (_.at "col1" * 10)
+    #
+    # `name`: Name of newly created column in result table.
+    # `f`: Function to apply on `self` table.
+    # `return`: `Table` value with `self` table and additionla column with provided name and values resulting from function `f` application.
```

Comment:
additional

Reply:
`self` with additional column `name` containing values returned by `f` for each row

Reply:
Also: "as a result of applying the function `f` to each row of the `self` table"

Reply:
Do we need to duplicate that much information? Return description is almost the same as function synopsis.